src/server.h

```diff
@@ -239,7 +239,8 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 	s32 playSound(ServerPlayingSound &params, bool ephemeral=false);
 	void stopSound(s32 handle);
 	void fadeSound(s32 handle, float step, float gain);
-	void stopAttachedSounds(u16 id);
+	// Stop all sounds attached to an object for a certain client
+	void stopAttachedSounds(session_t peer_id, u16 id);
```

Comment:
```suggestion
	void stopAttachedSounds(session_t peer_id, u16 object_id);
```

consistency between .h and .cpp